_scripts/clone-legal-docs.sh

```diff
@@ -130,25 +134,33 @@ copy_md() {
 
     log "Copying $md_name and parent locale directories into $MODULE_PATH/public/legal-docs/"
 
+    cd "$DOWNLOAD_PATH"
     results=()
     for src in **"/${md_name}.md"; do
         [ -f "$src" ] || continue
-        legal_docs_dir="$MODULE_PATH/public/legal-docs/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
+        legal_docs_dir="$LOCAL_PATH/$(dirname "$src" | sed -E 's/^.*\/([a-zA-Z_]+)\/.+/\1/; s/_/-/g')"
         [ -d "$legal_docs_dir" ] || mkdir -p "$legal_docs_dir"
         cp "$src" "$legal_docs_dir/$md_name.md"
         results+=("$(echo "$legal_docs_dir" | sed -E 's/.+\/([a-zA-Z_]+)/\1/; s/-/_/g')")
     done
 
-    log "Creating .json file containing array of available locales in $MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+    # log "Creating .json file containing array of available locales in $MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+    cat "$LOCAL_PATH/stats.json" | jq ".\"${md_name}.md\".locales" > "$MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+}
 
-    echo "[$(echo "${results[@]}" | tr ' ' ',' | sed -E 's/([^,]+)/"\1"/g')]" > "$MODULE_PATH/public/legal-docs/${md_name}_locales.json"
+copy_json() {
+    cd "$LOCAL_PATH"
+    curl -XGET https://raw.githubusercontent.com/mozilla/legal-docs/l10n_reference/.github/stats.json > stats.json
```

Comment:
For my own knowledge, what's `XGET` do for us here?

Reply:
Ah, since `curl` does a GET by default, I wondered if we explicitly needed it here. I don't mind either way, I just wasn't sure when we needed to specify it vs not.

Reply:
yeah it's probably not needed. i'll remove it.